resources/configure_dependencies.py

```diff
@@ -561,18 +565,42 @@ def InstallVTK(pkg: str, ver: str, gold_file: str):
     # Create envvars file
     module_file = open(module_file_name, "w")
 
+    petsc_dir = f"{install_dir}"
+    vtk_dir = f"{install_dir}"
+
     module_file.write(f'export CMAKE_PREFIX_PATH=$CMAKE_PREFIX_PATH:"{install_dir}"\n')
+    module_file.write(f'export PETSC_DIR={petsc_dir}\n')
+
+    if os.path.exists(f"{vtk_dir}/lib64"):
+        module_file.write(f'export LD_LIBRARY_PATH="{vtk_dir}/lib64":$LD_LIBRARY_PATH\n')
+    else:
+        module_file.write(f'export LD_LIBRARY_PATH="{vtk_dir}/lib":$LD_LIBRARY_PATH\n')
+
     module_file.close()
 
     ExecSub(f"chmod u+x {module_file_name}", log_file)
     log_file.close()
 
     print("\n########## OpenSn Dependency install complete ##########")
-    print("\nWhen compiling OpenSn the following environment variable" +
-          " may need to be set:\n")
+    print("\nWhen opening OpenSn in an IDE, the following environment variables" +
+          " need to be set:\n")
+
     print(f'CMAKE_PREFIX_PATH=$CMAKE_PREFIX_PATH:"{install_dir}"')
+    print(f'PETSC_DIR={petsc_dir}')
+    print()
+    print(TextColors.WARNING +
+          "When compiling OpenSn, in a terminal, the following environment "
+          "variables need to be set:\n" + TextColors.ENDC)
+
+    print(f'export CMAKE_PREFIX_PATH=$CMAKE_PREFIX_PATH:"{install_dir}"')
+    print(f'export PETSC_DIR="{petsc_dir}"')
+    if os.path.exists(f"{vtk_dir}/lib64"):
+        print(f'export LD_LIBRARY_PATH="{vtk_dir}/lib":$LD_LIBRARY_PATH')
+    else:
+        print(f'export LD_LIBRARY_PATH="{vtk_dir}/lib64":$LD_LIBRARY_PATH')
+
     print()
-    print(f"To set this environment variable automatically in a `bash` shell, execute:")
+    print(f"To set these terminal environment variables automatically, execute:")
     print(f"    $ source {module_file_name}\n")
```

Comment:
do we have modules? or do we require to use modules?

Reply:
Not in the sense you're talking about (which I'm guessing is environment modules). I'm not sure why Jan chose the name module, but I just restored the lines that I removed in my previous commit as is.

Reply:
while you are looking at the file, can you think of a better name? Since this took me down the wrong path, hopefully I am not the only one. Just a thought. Feel free to ignore.